content/en/docs/collector/architecture.md

```diff
@@ -0,0 +1,315 @@
+  it outside the Collector.
+
+The same receiver can be included in multiple pipelines and multiple pipelines
+can include the same exporter.
+
+## Pipelines
+
+A pipeline defines a path that data follows in the Collector: from reception, to
+processing (or modification), and finally to export.
+
+Pipelines can operate on three telemetry data types: traces, metrics, and logs.
+The data type is a property of the pipeline defined by its configuration.
+Receivers, processors, and exporters used in a pipeline must support the
+particular data type otherwise `ErrDataTypeIsNotSupported` will be reported when
+the configuration is loaded. A pipeline can be depicted the following way:
+
+```mermaid
+---
+title: Pipeline
+---
+flowchart LR
+    R1(Receiver 1) --> P1[Processor 1]
+    R2(Receiver 2) --> P1
+    RM(...) ~~~ P1
+    RN(Receiver N) --> P1
+    P1 --> P2[Processor 2]
+    P2 --> PM[...]
+    PM --> PN[Processor N]
+    PN --> FO((fan out))
+    FO --> E1[[Exporter 1]]
+    FO --> E2[[Exporter 2]]
+    FO ~~~ EM[[...]]
+    FO --> EN[[Exporter N]]
+
+    %% The stroke color matches the website header.
+    classDef default fill:#e3e8fc,stroke:#4f62ad
+```
+
+Pipelines can have one or more receivers. Data from all receivers is pushed to
+the first processor, which processes the data and then pushes it to the next
+processor (or a processor may drop the data if it is a “sampling” processor, for
+example), and so on until the last processor in the pipeline pushes the data to
+the exporters. Each exporter gets a copy of each data element. The last
+processor uses a `fanoutconsumer` to fan out the data to multiple exporters.
+
+The pipeline is constructed during Collector startup based on pipeline
+definition in the configuration.
+
+A pipeline configuration typically looks like this:
+
+```yaml
+service:
+  pipelines: # section that can contain multiple subsections, one per pipeline
+    traces: # type of the pipeline
+      receivers: [otlp, zipkin]
+      processors: [memory_limiter, batch]
+      exporters: [otlp, zipkin]
+```
+
+The above example defines a pipeline for the traces type of telemetry data,
+with three receivers, two processors, and three exporters.
+
+For details of config file format see
+[this document](https://docs.google.com/document/d/1NeheFG7DmcUYo_h2vLtNRlia9x5wOJMlV4QKEK05FhQ/edit#).
+
+### Receivers
+
+Receivers typically listen on a network port and receive telemetry data. Usually
+one receiver is configured to send received data to one pipeline. However, it is
+also possible to configure the same receiver to send the same received data to
+multiple pipelines. This can be done by listing the same receiver in the
+“receivers” key of several pipelines:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
+
+service:
+  pipelines:
+    traces: # a pipeline of “traces” type
+      receivers: [otlp]
+      processors: [memory_limiter, batch]
+      exporters: [jaeger]
```

Comment:
I count 11 more mentions of jaeger in the document, including several in diagrams (two of which I'm still working on).

Reply:
We do not need to remove all of them, only the instances where the jaeger exporter is used since it has been deprecated a while ago already, the receivers are still fine, but we still could consider using OTLP there. Jaeger can remain if named as a potential backend (which now supports OTLP)

Reply:
I changed mentions of jaeger exporter to otlp in 8e12f26.